node/service/Cargo.toml

```diff
@@ -140,17 +140,27 @@ substrate-test-runtime-client = { git = "https://github.com/paritytech/substrate
 
 
 [features]
-default = [ "kusama-native", "westend-native", "rococo-native" ]
+default = [
+	"kusama-native",
+	"westend-native",
+	"rococo-native",
+	"moonbase-runtime",
+	"moonriver-runtime",
+	"moonbeam-runtime"
+]
+
 kusama-native = [ "polkadot-service/kusama-native", "polkadot-cli/kusama-native" ]
 westend-native = [ "polkadot-service/westend-native", "polkadot-cli/westend-native" ]
 rococo-native = [ "polkadot-service/rococo-native", "polkadot-cli/rococo-native" ]
 
 test-spec = []
 
 runtime-benchmarks = [
+	"moonbeam-runtime",
 	"moonbeam-runtime/runtime-benchmarks",
+	"moonbase-runtime",
 	"moonbase-runtime/runtime-benchmarks",
```

Comment:
So if we are building with benchmarking, then we are building all the runtime? Is that the idea?

Why is moonriver not included here?

Reply:
Because it wasn't included before, so I didn't add it. It seems that we actually only use the moonbase runtime for our benchmarks.

Reply:
Done